src/hooks/screen/useMessageHook.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {Alert, NativeSyntheticEvent} from 'react-native';
+import {ContextMenuOnPressNativeEvent} from 'react-native-context-menu-view';
+import {useCallback, useContext} from 'react';
+
+import ChatSchema from '../../database/schema/ChatSchema';
+import ShareUtils from '../../utils/share';
+import UseMessageHook from '../../../types/hooks/screens/useMessageHook.types';
+import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+import {Context} from '../../context';
+import {setReplyTarget} from '../../context/actions/chat';
+
+function useMessageHook(): UseMessageHook {
+  const {localDb, refresh} = useLocalDatabaseHook();
+  const [replyPreview, dispatch] = (useContext(Context) as unknown as any).chat;
+
+  const setReplyPreview = useCallback((messageItem) => {
+    console.log('Reply Message');
+    setReplyTarget(messageItem, dispatch);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `console.log` statement on line 18 is likely used for debugging purposes and should be removed in the production code. It's a good practice to avoid console logs in production as they can expose sensitive information.

```diff
-   const setReplyPreview = useCallback((messageItem) => {
-     console.log('Reply Message');
-     setReplyTarget(messageItem, dispatch);
-   }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->